src/promptflow-tools/promptflow/tools/openai.py

```diff
@@ -111,8 +111,11 @@ def chat(
         seed: int = None,
         **kwargs
     ) -> [str, dict]:
-        chat_str = render_jinja_template(prompt, trim_blocks=True, keep_trailing_newline=True, **kwargs)
+        chat_str, escape_dict = render_jinja_template_wrapper(
```

Comment:
add comment to explain why need to escape and unescape.

Reply:
comment added for escape and unescape